css.md

```diff
@@ -88,7 +101,9 @@ The naming of functions, extends, mixins and variables should match the naming c
 ### File Naming
 All files that are pulled in via `@import` or `= require` should begin with an underscore (`_`) to follow the SASS/Sprockets convention that prevents these files from being included directly. Any file that is included directly with `stylesheet_link_tag` should not have an underscore and should live in the root `app/assets/stylsheets` directory.
 
-SASS file names should end in `.css.sass` to follow the Sprockets naming convention that defines what preprocessors to use in reverse order.
+SASS file names should end in `.sass` or - if necessary - `.sass.erb`.
+
+ > *Note:* `sass-rails` has [dropped support](https://github.com/sstephenson/sprockets/issues/643) of `.css.sass` which follows the [change in Sprockets naming convention](https://github.com/sstephenson/sprockets/commit/da2f1d176de6fae0bafc71e9cbf7805271856933) that defined what preprocessors to use in reverse order.
```

Comment:
You are correct, in fact both have dropped it. https://github.com/rails/sass-rails/pull/291